### PR TITLE
Pygame controls

### PR DESCRIPTION
**Pull request by sujith-rek**
## Summary

Part 4 of propsal,

pygame events,
need to complete remaining

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

(prefer Playwright recorded video or animated gif)

### CODE REVIEW

File: _sources/lectures/TWP60/TWP60_6.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
++ KEYUP
++ KEYDOWN
++ MOUSEMOTION
++ MOUSEBUTTONUP
++ MOUSEBUTTONDOWN
+
+KEYDOWN es un tipo de evento que se dispara cuando se presiona una tecla. como cuando presionas una tecla en el teclado.
+
+KEYUP es un tipo de evento que se activa cuando se suelta una tecla. como cuando quitas el dedo de una tecla.
+.. activecode:: ac_l60_6_en
```
Comment (NicolasSandoval) [inline _sources/lectures/TWP60/TWP60_6.rst:32]: Hi @sujith-rek! Please check that the Spanish version has a formatting problem.

**Review by NicolasSandoval — CHANGES_REQUESTED** [review]

File: _sources/lectures/TWP60/TWP60_6.rst (hunk 1 above)
Comment (sujith-rek) [reply _sources/lectures/TWP60/TWP60_6.rst:34]: Done

File: _sources/lectures/TWP60/TWP60_6.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    screen.fill(GREEN)
+
+    font=pygame.font.SysFont('timesnewroman',30)
+    left_key = font.render('se hace clic en el botón izquierdo del ratón,True,BLACK)
+    right_key = font.render('se hace clic en el botón derecho del ratón',True,BLACK)
+    mouse = font.render('El mouse se está moviendo',True,BLACK)
+    button_down = font.render('Se suelta el mouse',True,BLACK)
+    mouse_pos = font.render('Posición del mouse',True,BLACK)
+
+    def main():
```
Comment (NicolasSandoval) [inline _sources/lectures/TWP60/TWP60_6.rst:136]: @sujith-rek There is still an issue when running the `mouse` example.

**Review by NicolasSandoval — CHANGES_REQUESTED** [review]

File: _sources/lectures/TWP60/TWP60_6.rst (hunk 2 above)
Comment (sujith-rek) [reply _sources/lectures/TWP60/TWP60_6.rst:136]: sure. I'll look into it

Comment (sujith-rek) [reply _sources/lectures/TWP60/TWP60_6.rst:136]: its working in en, maybe i missed a space or unterminated string. Will resolve it

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**